src/Components/Navbar/Navbar.jsx

```diff
@@ -21,6 +25,7 @@ export const Navbar = () => {
         </Link>
         <Link
           to="/product-listing"
+          npm
```

Comment:
What does this npm do? seen this first time, so just curious.

Reply:
It is by mistake